src/context/viewer-context.tsx

```diff
@@ -3,11 +3,14 @@ import { Vault } from "@hyperion-framework/vault";
 
 export type ConfigOptions = {
   showTitle: boolean;
+  showIIIFBadge: boolean;
+  ignoreCaptionLabels: string[];
 };
 
 const defaultConfigOptions = {
   showTitle: true,
   showIIIFBadge: true,
+  ignoreCaptionLabels: [],
```

Comment:
Ah, well if you're setting a default value here, then my above comment might not matter.